app/src/main/java/org/oppia/app/player/state/CustomCheckbox.kt

```diff
@@ -0,0 +1,69 @@
+package org.oppia.app.player.state
+
+import android.content.Context
+import android.text.Html
+import android.text.Spannable
+import android.text.Spanned
+import android.view.View
+import android.widget.CheckBox
+import android.widget.CompoundButton
+import android.widget.TextView
+import android.widget.Toast
+import androidx.core.content.ContextCompat
+import org.oppia.app.R
+import org.oppia.util.data.UrlImageParser
+import android.content.res.ColorStateList
+import android.os.Build
+import android.view.Gravity
+
+// TODO(#163): Move this to a StateCardFragment Low-fi PR.
+/** Custom Checkbox for MultipleSelectionInputInteractionView. */
+class CustomCheckbox(context: Context, private val optionContents: String) : CheckBox(context) {
+
+  // Update default attributes of ItemSelectionInputInteractionView here.
+  init {
+    val paddingPixel = 2
+    val density = resources.displayMetrics.density
+    val paddingDp = (paddingPixel * density).toInt()
+    gravity = Gravity.LEFT
+    setTextColor(ContextCompat.getColor(context, R.color.oppiaDarkBlue))
```

Comment:
Can we set all of these properties (including the button tints) in a view-specific layout file, instead? I'm not keen on programmatically setting view properties because it makes them hard to find, especially when settings are split between code and XML. I think it would also largely simplify this & the next view since none of this extra initialization work needs to happen in code.

You may need to override the setText method, though, to make the HTML bit work correctly.

Ditto below for the other custom view.

Reply:
@BenHenning If we need to create recyclerview to show list of checkbox or radio button, there is no need for custom views right?

Reply:
Specified attributes in layout files for CustomCheckbox and CustomRadioButtons.
As we have used recyclerview for interactions and this holds checkbox and radiobuttons, Custom views are removed.